src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -6924,7 +6924,21 @@ def test_aks_create_with_monitoring_aad_auth_uai(self, resource_group, resource_
         aks_name = self.create_random_name('cliakstest', 16)
         self.create_new_cluster_with_monitoring_aad_auth(resource_group, resource_group_location, aks_name, user_assigned_identity=True)
 
-    def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_group_location, aks_name, user_assigned_identity=False):
+    @live_only()
```

Comment:
May I ask why these tests are marked as live only?

Reply:
these are based off of the two tests above which were also marked as @live_only()

Reply:
@pfrcks If these tests are marked as `@live_only()`, they cannot be rerun by our PR CI pipeline. 
It is suggested to investigate the reason why these tests are marked as `@live_only()`, and then solve those `@live_only()` tests in a separate new PR, do you think it is OK?

Reply:
sure, I will add a task to investigate this in our Dec sprint